### ISSUE TITLE
onnx-to-trt conversion fails for the CBAM structure with dynamic batchs

### ISSUE THREAD

**Issue by xingyueye** [NVIDIA/TensorRT#1652]
@pbridger @aaronp24 @ttyio @lukeyeager @elezar 
Hi~ Recently I encountered a strange problem, the onnx-to-trt conversion (either using trtexec or python API) still fails for models with the CBAM modules. I've pinpointed that error occurs when`out = avg_out + max_out` in ChannelAttention.
Can you help me with the confusion how does it occurs, and will the problem be corrected in the future?

The pytorch implementation of ChannelAttention likes that:
>class ChannelAttention(nn.Module):
>    def __init__(self, in_planes, ratio=16):
>       super(ChannelAttention, self).__init__()
>        self.avg_pool = nn.AdaptiveAvgPool2d(1)
>        self.max_pool = nn.AdaptiveMaxPool2d(1)
>        self.fc = nn.Sequential(nn.Conv2d(in_planes, in_planes // 16, 1, bias=False),
>                               nn.ReLU(),
>                               nn.Conv2d(in_planes // 16, in_planes, 1, bias=False))
>        self.sigmoid = nn.Sigmoid()
>    def forward(self, x):
>        avg_out = self.fc(self.avg_pool(x))
>        max_out = self.fc(self.max_pool(x))
>        out = avg_out + max_out
>        return self.sigmoid(out)

And I convert the dynamic batch onnx using:
>torch.onnx.export(model, x1, onnx_path, verbose=True,
>                      input_names=["input1"],
>                      output_names=["output"],
>                      dynamic_axes={'input1': {0: 'batch_size'},
>                                    'output': {0: 'batch_size'}})

**Comment by nvpohanh** [comment]
Also please share: which TRT version, which OS, which GPU, etc. Thanks

**Comment by nvpohanh** [comment]
Closing due to >14 days without activity. Please feel free to reopen if the issue still exists. Thanks
